properties.schema

```diff
@@ -48,6 +48,15 @@
                       "validators": [],
                       "help": "If enabled, the user's responses to questions will be saved and restored in each session. Note: this setting should be enabled in the majority of cases."
                     },
+                    "_shouldStoreAttempts": {
+                      "type": "boolean",
+                      "required": false,
+                      "default": false,
+                      "title": "Store question attempt states",
+                      "inputType": "Checkbox",
+                      "validators": [],
+                      "help": "If enabled, the user's attempt states to questions will be saved and restored in each session. Note: this setting should be disabled in the majority of cases."
```

Comment:
> Note: this setting should be disabled in the majority of cases.

Would be useful to have either here or in the README (and this setting needs adding to README too I think?) an explanation of when it might be useful to have this enabled.

Reply:
We don't have any use-cases for this at the moment. It will be for branching-style behaviour.